README.md

```diff
@@ -19,28 +19,28 @@ Packagist Stats was created by, and is maintained by [Graham Campbell](https://g
 
 This project is relatively small, and it's usage is pretty simple.
 
-Install dependencies:
+Installation via composer:
 
 ```bash
-$ composer install --no-dev
+$ composer global require graham-campbell/packagist-stats dev-master
```

Comment:
@GrahamCampbell 
The package is not available in a stable-enough version according to your minimum-stability setting.
How do you suggest?